release/release_options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "read_from_cache": false
```

Comment:
Actually this can be useful if things go wrong during a release, not that that ever happens of course.